### DISCUSSION TITLE
`sf package install` - Supress email notification

### DISCUSSION THREAD

**Discussion by pkozuchowski** [forcedotcom/cli#2874] [Ideas]
Would it be possible to add a flag to supress email notification for installed packages?
My rationale is that installing packages on scratch orgs generates a lot of email noise from the installed packages.

**Comment by VivekMChawla** [comment]
@pkozuchowski - Thank you for the suggestion.

I checked the Tooling and Metadata APIs for options to suppress package install email notifications on an install-by-install basis. These APIs do not currently support this.

I also checked if there was a setting to suppress package install email notifications at the org level, but could not find one.

Unfortunately, without MDAPI/Tooling API support or an org-wide setting exposed as a scratch org config value, the functionality you're requesting can't be delivered.  

If you feel strongly about this feature request, please consider raising it directly with the Packaging team by posting to the [Managed Packages group](https://partners.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F93A000000HXAq) (Partner Community) or the [Unlocked Packages group](https://trailhead.salesforce.com/trailblazer-community/groups/0F93A000000Lg5USAS?tab=discussion&sort=LAST_MODIFIED_DATE_DESC) (Trailblazer Community).
